zerver/lib/actions.py

```diff
@@ -2441,15 +2441,15 @@ def do_update_user_presence(user_profile, client, log_time, status):
 
 def update_user_activity_interval(user_profile, log_time):
     # type: (UserProfile, datetime.datetime) -> None
-    event={'user_profile_id': user_profile.id,
+    event = {'user_profile_id': user_profile.id,
            'time': datetime_to_timestamp(log_time)}
     queue_json_publish("user_activity_interval", event,
                        lambda e: do_update_user_activity_interval(user_profile, log_time))
 
 def update_user_presence(user_profile, client, log_time, status,
                          new_user_input):
     # type: (UserProfile, Client, datetime.datetime, int, bool) -> None
-    event={'user_profile_id': user_profile.id,
+    event = {'user_profile_id': user_profile.id,
            'status': status,
            'time': datetime_to_timestamp(log_time),
            'client': client.name}
```

Comment:
For these cases where one is defining a multi-line dict object, you need update the line-wrapping for the following lines.

Can you update this here (and in the other similar cases in this pull request)?